pkg/services/mssql/plan_details.go

```diff
@@ -22,6 +22,11 @@ type planDetails interface {
 	getTierProvisionParameters(
 		service.Instance,
 	) (map[string]interface{}, error)
+	getTierUpdateParameters(
+		service.Instance,
+	) (map[string]interface{}, error)
+	getUpgradeSchema() *service.InputParametersSchema
```

Comment:
Nit: I think elsewhere, we use the term "update" rather than "upgrade." Can we change this for consistency? Or am I overlooking a reason for "upgrade" being used here?

Reply:
Yeah, can do.